packages/react/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -44,6 +44,7 @@ const ButtonGroup = ({ buttons, enableSizeByContent }) => {
               ),
               item => {
                 item.style.width = `${width + 1}px`;
+                item.style.display = `inline-block`;
```

Comment:
Given `display: inline-block` doesn't need JavaScript-based calculation, would you move this to `_buttongroup.scss`?